nefertari/utils/data.py

```diff
@@ -94,3 +94,43 @@ def obj2dict(obj, classkey=None):
         return data
     else:
         return obj
+
+
+class FieldData(object):
```

Comment:
We started using this class after we needed to have an access to field's init kwargs inside processors.
I think it's more pleasant to deal with one object that has properties, than passing 3 args instead of it.